doc/adding-new-regression-tests.md

```diff
@@ -0,0 +1,71 @@
+# Adding new regression tests
+
+1. Update the flow to replace any single line conditionals around `Phraselist`s with multiple line conditionals. This is so that we get useful information from the running the coverage utility. Single line conditionals will show up as having been exercised irrespective of whether they caused something to be added to the `Phraselist`.
+
```

Comment:
Not exactly related to this PR, but I've skimmed the codebase ([using GH search](https://github.com/alphagov/smart-answers/search?utf8=%E2%9C%93&q=Phraselist)) and it looks like Phraselists are nearly extinct. I think it would make sense to look at removing it as a feature.